graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/FieldTypeCache.java

```diff
@@ -91,51 +91,20 @@ private ColumnType normalize(ColumnType type) {
 
   protected GraphQLScalarType getScalar(Type type) {
```

Comment:
Why are all the types being removed here? Isn't cql-first staying the same?

Reply:
Nothing changed, the logic was just moved to `CqlScalar.fromCqlType(type)` (which is possible because `CqlScalar` was turned into an enum).

Reply:
Ah, gotcha.